launcher/android/BUILD.bazel

```diff
@@ -7,15 +7,15 @@ android_library(
         "//src/main/java/com/code_intelligence/jazzer/android:jazzer_standalone_android.apk",
     ],
     tags = ["manual"],
-    target_compatible_with = SKIP_ON_WINDOWS,
+    target_compatible_with = ANDROID_ONLY,
 )
 
 android_binary(
     name = "jazzer_android",
     manifest = ":android_manifest",
     min_sdk_version = 26,
     tags = ["manual"],
-    target_compatible_with = SKIP_ON_WINDOWS,
+    target_compatible_with = ANDROID_ONLY,
```

Comment:
Since this is a top-level android_binary target, does it need this restriction? Without it, we wouldn't have to worry about this being run in CI.

Reply:
android_binary wont build on Windows, so it's good to have the SKIP_ON_WINDOWS, do you think I should change it back to that?

Reply:
That's a good idea, please do.